lib/test.js

```diff
@@ -70,6 +70,9 @@ Object.keys(assert).forEach(function (el) {
 	};
 });
 
+Test.prototype._capt = assert._capt
+Test.prototype._expr = assert._expr
```

Comment:
@uiureo Hmm, this workaround is required for now, but looks a bit messy.
To clear this up, I have to create a module to enhance prototype, not function nor object.
Would you wait some more days?